src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {'My name is '}
+        {name}
+      </h2>
+      { age
+        && (
+        <p className="Person__age">
+          {'I am '}
+          {age}
+        </p>
+        )}
```

Comment:
add indents for nested elements and fix your formatting
```suggestion
      {age && (
        <p className="Person__age">
          {'I am '}
          {age}
        </p>
      )}
```

Reply:
fixed all